packages/app/src/app/overmind/namespaces/editor/internalActions.ts

```diff
@@ -338,7 +322,27 @@ export const forkSandbox: AsyncAction<{
       });
     }
 
-    await actions.internal.setCurrentSandbox(forkedSandbox);
+    // When we have a server we want to set a brand new sandbox,
+    // as the preview also needs to be managed correctly
+    if (getTemplateDefinition(forkedSandbox.template).isServer) {
+      await effects.vscode.closeAllTabs();
```

Comment:
Is this really needed for servers?

Reply:
Oh yeah, fixed this now. Does not seem we need it! 👏